src/w3c/headers.js

```diff
@@ -634,11 +634,23 @@ export function run(conf, doc, cb) {
   conf.isPR = conf.specStatus === "PR";
   conf.isPER = conf.specStatus === "PER";
   conf.isMO = conf.specStatus === "MO";
+  conf.isNote = conf.specStatus === "NOTE";
```

Comment:
Looking at the [documentation](https://github.com/w3c/respec/wiki/specStatus), shouldn't the status be `WG-NOTE`?

Reply:
You are correct, there is also FPWD-NOTE, it seems. I still don't know why we have "NOTE"? It seems to be "a thing" - it has an associated W3C Style? Should we kill that?